chcemvediet/apps/inforequests/views/inforequest.py

```diff
@@ -55,8 +55,16 @@ def inforequest_mine(request):
             .order_by_pk()
             .select_related(u'obligee')
             )
-    closed_inforequests = (Inforequest.objects
-            .closed()
+    answered_inforequests = (Inforequest.objects
```

Comment:
@martinmacko47 Navrhoval by som premenovat aj `inforequests` na `pending_inforequests`

Reply:
:ok:

Reply:
@viliambalaz Som zmateny, preco si premenoval `inforequests` na `pending_inforequests` aj v `obligee_action_dispatcher` viewe? Ved to nijak nesuvisi s tabulkou inforequestov, ci? Navyse tam ine inforequesty ako tie pending ani nemame, takze zbytocne rozlisovat. Zase nie je dobre teraz vsade v kode volat vsetky premenne `pending_inforequests`, aj ked je jasne, ze dana cast kodu pracuje iba s neuzatvorenymi. Pls premenuj to iba v tomto view s tabulkami a v suvisiacich vediach.

Reply:
V ramci view tam bola iba jedna premenna inforequest, ale chcel som to rozlisit v ramci celeho suboru. Aj pre zverejnene inforequesty vo view `inforequest_index` som nazval premennu `published_inforequests`, tiez tam ine inforequesty neboli.
:ok: Premenoval som to späť.